### PR TITLE
Update system console paths

### PR DESCRIPTION
**Pull request by esethna**
Updating the paths to all the system console settings after the reorg

### CODE REVIEW

File: source/help/messaging/attaching-files.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Mattermost has a built in file previewer that is used to view media, download fi
 #### Sharing Public Links
 Public links allow you to share file attachments with people outside your Mattermost team. Open the file previewer by clicking on the thumbnail of an attachment, then click **Get Public Link**. This opens a dialog box with a link to copy. When the link is shared and opened by another user, the file will automatically download. 
 
-If **Get Public Link** is not visible in the file previewer then contact your system administrator to enable public file links in the System Console file settings.
+If **Get Public Link** is not visible in the file previewer then contact your system administrator to enable public file links in the System Console security settings.
```
Comment (it33) [inline source/help/messaging/attaching-files.md]: Why not **System Console** > **Security** settings?

File: source/deployment/push.rst [hunk 2]
```diff
@@ -99,22 +99,22 @@ To setup HPNS please follow the following steps:
...
-     2. Have "Account A" close the mobile application, but do not log out. The mobile app needs to be in the background for the test to work. Make sure "Account A"does not have Mattermost open in any other web, desktop or mobile app for at least 30 seconds in order to make the account go offline. 
-     3. Using "Account B", on the same Mattermost team as "Account A", use the "More" menu under the Direct Messages section in the left hand side of the team site to add "Account A" to the Direct Message list. Confirm from the indicator next to "Account A"'s name in the direct message list that "Account A" is offline. 
+     1. From the mobile application used by "Account A", click the three dot menu on the top right and go to **Account Settings** > **Notifications** > **Mobile push notifications**. Click "Edit" and select "For mentions and direct messages", then "Save" the setting.
```
Comment (it33) [inline source/deployment/push.rst]: When do we say "Edit" and "Save" vs. **Edit** and **Save**?

File: source/install/ee-prod-rhel-6.rst [hunk 3]
```diff
@@ -323,7 +323,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
+4. Update **Notification** > **Email** settings. We recommend using an email sending service.
```
Comment (it33) [inline source/install/ee-prod-rhel-6.rst]: The second sentence seems weird--are we giving and instruction or a recommendation? 

Propose instead `Update **Notification** > **Email** settings to setup an SMTP email service.`

File: source/install/ee-prod-ubuntu.rst [hunk 4]
```diff
@@ -302,9 +302,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
-   The example shows how an Amazon SES setup might look (sample
-   credentials shown below are not real).
+4.  Update **Notification** > **Email** settings. We recommend using an email sending service.The example below assumes AmazonSES.
```
Comment (it33) [inline source/install/ee-prod-ubuntu.rst]: Same feedback as: https://github.com/mattermost/docs/pull/238/files#r67271976

File: source/install/prod-debian.rst [hunk 5]
```diff
@@ -474,8 +474,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
-   The example below assumes AmazonSES.
+4.  Update **Notification** > **Email** settings. We recommend using an email sending service. The example below assumes AmazonSES.
```
Comment (it33) [inline source/install/prod-debian.rst]: same feedback as https://github.com/mattermost/docs/pull/238/files#r67271976

File: source/install/prod-rhel-6.rst [hunk 6]
```diff
@@ -322,8 +322,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
-   The example below assumes AmazonSES.
+4.  Update **Notification** > **Email** settings. We recommend using an email sending service. The example below assumes AmazonSES.
```
Comment (it33) [inline source/install/prod-rhel-6.rst]: same feedback as https://github.com/mattermost/docs/pull/238/files#r67271976

File: source/install/prod-rhel-7.rst [hunk 7]
```diff
@@ -328,8 +328,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
-   The example below assumes AmazonSES.
+4.  Update **Notification** > **Email** settings. We recommend using an email sending service. The example below assumes AmazonSES.
```
Comment (it33) [inline source/install/prod-rhel-7.rst]: same feedback as https://github.com/mattermost/docs/pull/238/files#r67271976

File: source/install/prod-ubuntu.rst [hunk 8]
```diff
@@ -301,9 +301,7 @@ Finish Mattermost Server setup
    ``system_admin`` role, which gives you access to the System Console.
 3. From the ``town-square`` channel click the dropdown and choose the
    ``System Console`` option
-4. Update Email Settings. We recommend using an email sending service.
-   The example shows how an Amazon SES setup might look (sample
-   credentials shown below are not real).
+4.  Update **Notification** > **Email** settings. We recommend using an email sending service. The example below assumes AmazonSES.
```
Comment (it33) [inline source/install/prod-ubuntu.rst]: same feedback as https://github.com/mattermost/docs/pull/238/files#r67271976

File: source/install/troubleshooting.rst [hunk 9]
```diff
@@ -86,7 +86,7 @@ This error may appear in server logs when attempting to sign-up when using self-
 
 **Solution:** Set up a load balancer like Ngnix `per production install guide <http://docs.mattermost.com/install/prod-debian.html#set-up-nginx-with-ssl-recommended>`__. The core team is looking into allowing self-signed certificates in the future.
```
Comment (it33) [inline source/install/troubleshooting.rst]: Nginx should be NGINX

**Comment by esethna** [comment]
Thanks, tweaks made! Merging now.

**Pull request merged**